### DISCUSSION TITLE
K8s Cluster Unreachable.

### DISCUSSION THREAD

**Discussion by Ehsaan-Azizi** [k3s-io/k3s#3540] [Q&A]
hi guys,
I am trying to deploy an image to our IKS using github actions workflow and Helm charts but it is failing with the below error. Any ideas?? 

Error: Kubernetes cluster unreachable: Get "http://localhost:8080/version?timeout=32s": dial tcp [::1]:8080: connect: connection refused

* I am not using it locally on my laptop so not sure why it is pointing to localhost?? 
*  I have checked my kubeconfig file and everything is looking as they are supposed to.

**Comment by brandond** [comment]
1. What is IKS?
2. The kubectl packaged with k3s has been modified to load kubeconfig from a location that might not be supported by other tools. You might try running `export KUBECONFIG=/etc/rancher/k3s/k3s.yaml` before running helm.

**Comment by Ehsaan-Azizi** [comment]
IKS = IBM K8s Services. Btw, where should I run that export KUBECONFIG=/etc/rancher/k3s/k3s.yaml command and what would be the next step after exporting it? I am on a Windows OS.

**Comment by brandond** [comment]
Helm needs to know where to find the kubeconfig that contains credentials and configuration for accessing your cluster. I don't think my instructions will help you if you're not using K3s on Linux. Can I ask why you're posting here if you're not using K3s?
